### DISCUSSION TITLE
Can a project using nodejs native addon be listed on the Apple Store?

### DISCUSSION THREAD

**Discussion by CaryTrivett** [nodejs-mobile/nodejs-mobile#29] [Q&A]
I want to use sqlite3 in the nodejs of the project.

**Comment by staltz** [comment]
Yes.

There are known iOS apps currently on the Apple App Store using nodejs-mobile, such as [Manyverse](https://manyver.se/). This app doesn't use sqlite3, but it uses a number of native addons, such as sodium-native, leveldown, and others.
